.github/workflows/release.yml

```diff
@@ -170,7 +170,7 @@ jobs:
       - name: Setup Node.js
         uses: actions/setup-node@v1
         with:
-          node-version: 10.x
+          node-version: 14
 
       - name: Download Cached Deps
```

Comment:
Okay, it won't let me add a change suggestion to the unedited part of the file. But we should remove this cache step

Reply:
@aswanson-nr riiiight i should have asked what you meant by that 🏚️ Before I change this on all the rest is my update correct?